frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -469,7 +469,9 @@ func dispatch(d *dispatchState, cmd command, opt dispatchOpt) error {
 		err = dispatchCopy(d, c.SourcesAndDest, opt.buildContext, true, c, "", opt)
 		if err == nil {
 			for _, src := range c.Sources() {
-				d.ctxPaths[path.Join("/", filepath.ToSlash(src))] = struct{}{}
+				if !strings.HasPrefix(src, "http://") && !strings.HasPrefix(src, "https://") {
```

Comment:
perhaps filter on `://` (which should exclude anything with a scheme)?

Reply:
No other scheme should be supported anyway and this is not the best place to do validation.

Reply:
Should we take into account that schemes are case-insensitive? https://tools.ietf.org/html/rfc2616#section-3.2.3

Reply:
Thats true but probably for another issue(this code is only for the filter aspect). I checked that this does not with docker with or without buildkit.